src/static/css/index.css

```diff
@@ -116,7 +116,8 @@ p {
   width: 86vw;
 }
 
-.intro-image-2019-wrapper {
+.intro-image-2019-wrapper,
+.intro-image-2020-wrapper {
```

Comment:
WDYT about renaming this class to `.intro-image-wrapper` for year-agnostic styles?

Reply:
Yes, but I would wait and do this part of #894 because at that point it is one wrapper.

Reply:
Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/893/commits/b5351ed6b55a2e02432f50f99765910e1c794ae9